linera-sdk/src/views/system_api.rs

```diff
@@ -80,7 +83,23 @@ impl KeyValueStore {
 }
 
 impl WithError for KeyValueStore {
-    type Error = ViewError;
+    type Error = StoreError;
+}
+
+/// The error type for [`ViewContainer`] operations.
+#[derive(Error, Debug)]
```

Comment:
I think it's a good idea to use `#[non_exhaustive]` for public error enums so that we can extend them without it being a breaking change.

Reply:
Should we create a "good first task" ?